dcm2bids/sidecar.py

```diff
@@ -152,13 +172,17 @@ def isLink(self, data, criteria):
         """
 
         def compare(name, pattern):
-            if self.searchMethod == "re":
-                return bool(re.search(pattern, str(name)))
+            if not self.caseSensitive:
```

Comment:
What do I do if I want *some* criteria to be case sensitive and others not?

Reply:
No I don't want to over complicated dcm2bids.